docs/index.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: home
+
+# Header Content
+# =================================
+header:
+    - logo_image: assets/img/dsa_logo.svg
+
+# Banner Content
+# =================================
+title: Harness the full potential of your digital pathology data
+subtitle: A containerized web-based platform for the analysis, visualization, management and annotation of whole-slide digital pathology imaging data
+
+hero_image: assets/img/dsa_hero.jpg
+
+hero_buttons:
+  - link_text: View Demo
+    link: #
+
+  - link_text: Read Docs
+    link: #
+
+  - link_text: View on GitHub
+    link: #
+
+# What is DSA Section
+# =================================
+about_title: What is DSA?
+about_text: The Digital Slide Archive (DSA) is a platform  that provides the ability to store, manage, visualize and annotate large imaging data sets.  The DSA is an umbrella term and consists of an analysis toolkit (HistomicsTK), a library to manage/visualize slides (HistomicsUI) as well as a database layer (using Mongo), and a web-server that provides a rich API.  This system allows;
+
+# Capabilities of DSA
+about_list:
+  - list_item: Organize images from a variety of assetstores, such as local files systems and S3.
+  - list_item: Provide user access controls.
+  - list_item: Image annotation and review.
+  - list_item: Run algorithms on all or parts of images.
+
+# Resource links
+about_resources:
+  - name: Gitter Channel
+    link: #
+    icon: assets/img/resource-icons/icon_gitter.png
+
+  - name: Discourse Forum
+    link: #
+    icon: assets/img/resource-icons/icon_discourse.png
+
+  - name: JupyterNotebooks
+    link: #
+    icon: assets/img/resource-icons/icon_jupyternb.png
+
+  - name: Youtube Channel
+    link: #
+    icon: assets/img/resource-icons/icon_youtube.png
+
+# Platforms Content
+# =================================
+platforms:
+  # HistomicsUI
+  - title: HistomicsUI
+    description: Web-based application for examining, annotating, and processing histology images to extract both low and high level features (e.g. cellular structure, feature types).
```

Comment:
Let's turn this into a complete sentence: `HistomicsUI is a web-based ...`

Reply:
Fixed in 51f03b5f5d244cd83cc6acb83fbdc3cba4987b7d